src/rollback_to_stable/rts_global.c

```diff
@@ -0,0 +1,151 @@
+/*-
+ * Copyright (c) 2014-present MongoDB, Inc.
+ * Copyright (c) 2008-2014 WiredTiger, Inc.
+ *	All rights reserved.
+ *
+ * See the file LICENSE for redistribution information.
+ */
+
+#include "wt_internal.h"
+
+/*
+ * __wt_rts_global_check --
+ *     Check to the extent possible that the rollback request is reasonable.
+ */
+int
+__wt_rts_global_check(WT_SESSION_IMPL *session)
```

Comment:
The global name is a bit confusing. May be just __wt_rts_check

Reply:
I've made the change you suggested. The `global` name is a prefix that matches the grouping in the source file, so every function in this source file should be __rts_global_<rest_of_name>, which gives some way of seeing the grouped functionality together. It didn't make much sense in this file, since `global` wasn't a very good prefix. After chatting with Will we decided the code in here was really top-level RTS implementation, so I renamed the file to rts.c instead.